src/masternodes/poolpairs.h

```diff
@@ -183,10 +183,10 @@ class CPoolPair : public CPoolPairMessage
         return onReclaim(address, resAmountA, resAmountB);
     }
 
-    Res Swap(CTokenAmount in, PoolPrice const & maxPrice, std::function<Res(CTokenAmount const &)> onTransfer);
+    Res Swap(CTokenAmount in, PoolPrice const & maxPrice, std::function<Res(CTokenAmount const &)> onTransfer, bool postBayfrontGardens = false);
```

Comment:
should the default be `true` instead? It doesn't matter much, but the default should be what's more likely to be more common, which in this case would be `true`.